tests/beaker_tests/cisco_bgp_neighbor_af/test_bgpneighboraf.rb

```diff
@@ -67,12 +68,10 @@
 # Top-level keys set by caller:
 # tests[:master] - the master object
 # tests[:agent] - the agent object
-# tests[:show_cmd] - the common show command to use for test_show_run
 #
 tests = {
```

Comment:
This is gonna be *fun* to merge with the changes from #176 - but we'll burn that bridge when we come to it. :persevere: